### PR TITLE
[Docs] Translate Project page to Spanish

### LINKED ISSUE #1703: [Docs] Translate Project page to Spanish
**Current State:**
The actual documentation is in english.

**Desired State:**
I want to translate this page:https://meshery.layer5.io/docs/project to spanish.

---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by homero304**
Signed-off-by: homero304 <homeromaring@gmail.com>

**Description**

This PR fixes #1703 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

Please review my PR

### CODE REVIEW

File: docs/pages/es/project/CONTRIBUTING-gitflow.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Trabajando por Fork
+Simplemente dirígete a la página de GitHub y haz clic en el botón "Fork". Es así de simple. Una vez que hayas hecho eso, puedes usar tu cliente git favorito para clonar tu repositorio o simplemente dirigirse directamente a la línea de comando:
+
+## Clona tu fork en tu máquina local
+```
+git clone git@github.com:USERNAME/FORKED-PROJECT.git
+```
+Mantener Tu Fork Actualizado
+Si bien este no es un paso necesario, si planeas hacer algo más que una pequeña solución rápida, querrás asegurarte de mantener tu fork actualizado rastreando el repositorio "upstream" original que bifurcaste. Para hacer esto, deberás agregar un "remote":
```
Comment (jesuslerma) [inline docs/pages/es/project/CONTRIBUTING-gitflow.md:9]: Rename "original que bifurcaste" to "original al que hiciste fork" it's not recommended to transalte "fork" to "bifurcación". Also during the rest of the text you're using the word: fork. so keep it :+1:

File: docs/pages/es/project/CONTRIBUTING-gitflow.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+## Agregar repositorio 'upstream' a la lista de "remotes"
```
Comment (jesuslerma) [inline docs/pages/es/project/CONTRIBUTING-gitflow.md:11]: Change "a la lista de remotes" to "a la lista de remotos"

File: docs/pages/es/project/CONTRIBUTING-gitflow.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+```
+git remote add upstream https://github.com/layer5io/meshery.git
+```
+("meshery" se usa como repositorio de ejemplo. Asegúrate de hacer referencia al repositorio _actual_ al que está contribuyendo, por ejemplo, "meshery-linkerd").
+
+## Verifica el nuevo "remote" llamado 'upstream'
```
Comment (jesuslerma) [inline docs/pages/es/project/CONTRIBUTING-gitflow.md:17]: Change "Verifica el nuevo remote" to "Verifica el nuevo repositorio remoto"

File: docs/pages/es/project/CONTRIBUTING-gitflow.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+```
+git remote -v
+```
+Siempre que quieras actualizar tu fork con los últimos cambios de 'upstream', primero deberás buscar las ramas del repositorio 'upstream' y los últimos "commits" para llevarlos a tu repositorio:
+
+## Obtener desde el "remote" 'upstream'
```
Comment (jesuslerma) [inline docs/pages/es/project/CONTRIBUTING-gitflow.md:23]: Change "remote" to "remoto"

File: docs/pages/es/project/CONTRIBUTING-gitflow.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+```
+git checkout master
+git merge upstream/master
+```
+Si no hay "commits" únicos en la rama "master" local, git simplemente realizará un avance rápido. Sin embargo, si has realizado cambios en "master" (en la gran mayoría de los casos, probablemente no deberías hacerlo; consulta la siguiente sección, es posible que tengas que lidiar con conflictos. Al hacerlo, ten cuidado de respetar los cambios realizados anteriormente.
+
+Ahora, tu rama "master" local está actualizada con todo lo modificado en 'upstream'.
+
+**Crear una "Branch"** (haciendo tu trabajo)
+Siempre que comiences a trabajar en una nueva función o corrección de "bugs", es importante que crees una nueva rama. No solo es un flujo de trabajo de git adecuado, sino que también mantiene tus cambios organizados y separados de la rama "master" para que puedas enviar y administrar fácilmente múltiples "pull requests" para cada tarea que completes.
```
Comment (jesuslerma) [inline docs/pages/es/project/CONTRIBUTING-gitflow.md:44]: Don't translate "fast-forward". So change "realizará un avance rápido" to "realizara un fast-forward".

finish this paragraph with a closing )

File: docs/pages/es/project/security-vulnerabilities.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: Vulnerabilidades de seguridad
+permalink: es/project/security-vulnerabilities
+description: Cómo el equipo de Meshery maneja las vulnerabilidades de seguridad.
+---
+
+# Informar una vulnerabilidad
+Estamos muy agradecidos con los investigadores de seguridad y los usuarios
```
Comment (jesuslerma) [inline docs/pages/es/project/security-vulnerabilities.md:9]: Change 'Estamos muy agradecidos con los investigadores de seguridad y los usuarios que informan sobre las vulnerabilidades de seguridad de Meshery' to
'Estamos muy agradecidos con quienes investigan temas de seguridad y quienes informan sobre las vulnerabilidades de seguridad de Meshery.'

**Review by jesuslerma — COMMENTED** [review]
Hi thanks for your effort on this PR @homero304  please do the following requested changes. It will help you to practice the contribution workflow :)

**Review by jesuslerma — APPROVED** [review]
You did an excelent job here @homero304  thanks! LGTM

**Comment by leecalcote** [comment]
Niiiice!

**Review by leecalcote — APPROVED** [review]
Thank you, @homero304 💪

**Pull request merged**